connector/signaltometrics/testdata/traces/exponential_histograms/config.yaml

```diff
@@ -0,0 +1,32 @@
+signaltometrics:
+  spans:
+    - name: with_resource_foo_only
+      description: Spans with resource attribute white listing for resource.foo as a exponential histogram metric
+      include_resource_attributes:
+        - key: resource.foo
+      exponential_histogram:
+        value: Milliseconds(end_time - start_time)
+    - name: with_custom_count
+      description: Spans with custom count OTTL expression as a exponential histogram metric
+      exponential_histogram:
+        value: Milliseconds(end_time - start_time)
+        count: "2" # count each span twice
+    - name: http.trace.span.duration
+      description: Span duration for HTTP spans as a exponential histogram metric
+      attributes:
+        - key: http.response.status_code
```

Comment:
Checking my understanding of how this is working. This implicitly filters events that have a value for this attribute. Events that don't have a value are ignored and neither the count nor the value is used. But when a default_value is supplied, there's no filtering. Is my understanding correct?

Reply:
> But when a default_value is supplied, there's no filtering

There is still filtering but if the key is not found in the attributes then the default value is used. For example, for the following definition of an attribute `attr.foo`:

```
- key: attr.foo
   default_value: bar
```

and events as:

```
event1:
  attributes:
    - key: attr.foo
       value: foo1
  ...
event2:
  attributes:
    - key: attr.foo
       value: foo2
  ...
event3:
  attributes: []
```

There will be 3 datapoints:

```
dp1: # for event 1
  attributes:
    - key: attr.foo
       value: foo1
dp2: # for event 2
  attributes:
    - key: attr.foo
       value: foo2
dp3: # for event 3 with attr.foo set to the default value of `bar`
  attributes:
    - key: attr.foo
       value: bar
```

Examples are also available in `testdata/traces/{metric_type}/{config.yaml, output.yaml}` (the input file is `testdata/traces/traces.yaml`